evaluator/evaluator_threads_manager.py

```diff
@@ -31,11 +32,12 @@
 class EvaluatorThreadsManager:
```

Comment:
Is it possible to rename this class (and name) with task ?

Reply:
+1